lib/watir_drops/page_object.rb

```diff
@@ -34,36 +52,40 @@ def elements(name, &block)
         element_list << name.to_sym
       end
 
-      def element(name, &block)
+      def element(name, required: false, &block)
         define_method(name) do |*args|
           self.instance_exec(*args, &block)
         end
 
         define_method("#{name}=") do |val|
           watir_element = self.instance_exec &block
           case watir_element
-          when Watir::Radio
-            watir_element.set if val
-          when Watir::CheckBox
-            val ? watir_element.set : watir_element.clear
-          when Watir::Select
-            watir_element.select val
-          when Watir::Button
-            watir_element.click
-          when Watir::TextField, Watir::TextArea
-            watir_element.set val if val
-          else
-            watir_element.click if val
+            when Watir::Radio
+              watir_element.set if val
```

Comment:
Do not increase the indentation of these lines. https://github.com/bbatsov/ruby-style-guide#indent-when-to-case